src/main/java/io/temporal/worker/WorkerOptions.java

```diff
@@ -143,8 +144,8 @@ public Builder setMaxConcurrentLocalActivityExecutionSize(
      * every 10 seconds. This can be used to protect down stream services from flooding. The zero
      * value of this uses the default value. Default is unlimited.
      */
-    public Builder setTaskQueueActivitiesPerSecond(double taskQueueActivitiesPerSecond) {
-      this.taskQueueActivitiesPerSecond = taskQueueActivitiesPerSecond;
+    public Builder setMaxTaskQueueActivitiesPerSecond(double maxTaskQueueActivitiesPerSecond) {
```

Comment:
why not have negative check here? Or do we treat that as 0? Interesting that we have different behavior than above (although I guess w/different defaults that makes sense)

Reply:
The above option is used only locally. So validation should be done by the SDK. The maxTaskQueueActivitiesPerSecond is an option sent to the service. We made a design decision to not validate on the client any server-side options.